java/src/com/ibm/streamsx/topology/internal/context/ZippedToolkitStreamsContext.java

```diff
@@ -99,9 +99,8 @@ public Path pack(final Path folder, String namespace, String name, String tkName
 		paths.add(folder);
 		
 		addAllToZippedArchive(paths, zipFilePath);	
-		JSONObject jso = new OrderedJSONObject();
-		super.deleteToolkit(new File(paths.get(1).toString()), jso);
-		super.deleteToolkit(new File(paths.get(2).toString()), jso);
+		new File(paths.get(1).toString()).delete();
+		new File(paths.get(2).toString()).delete();
```

Comment:
Minor point but just realised it's much simpler:

`paths.get(1).toFile().delete()`